hail/src/main/scala/is/hail/cxx/package.scala

```diff
@@ -69,6 +69,7 @@ package object cxx {
       case _: TBinary => s"store_address($a, $v)"
       case _: TArray => s"store_address($a, $v)"
       case _: TBaseStruct => s"memcpy($a, $v, ${ pType.byteSize })"
+      case TVoid => s"store_int($a, $v)"
```

Comment:
Same comment here---I'd prefer to explicitly break out the TVoid case in the entry/exit points of the compiled function, since the behavior we need to make it work right now isn't strictly "correct" in terms of void representation.